docker.go

```diff
@@ -34,11 +34,12 @@ type (
 
 	// Login defines Docker login parameters.
 	Login struct {
-		Registry string // Docker registry address
-		Username string // Docker registry username
-		Password string // Docker registry password
-		Email    string // Docker registry email
-		Config   string // Docker Auth Config
+		Registry    string // Docker registry address
+		Username    string // Docker registry username
+		Password    string // Docker registry password
+		Email       string // Docker registry email
+		Config      string // Docker Auth Config
+		AccessToken string // OIDC Access Token
```

Comment:
can we be consistent with naming, and add oidc to the variable name

Reply:
I kept it generic as access token because that's what it is... we might get an access token via a different flow in AWS?